### PR TITLE
Update Adopt a Widget contributions list

### LINKED ISSUE #5286: 'Adopt a Widget' page issue
Page URL: https://flutter.dev/adoptawidget/
Page source: https://github.com/flutter/website/tree/master/src/adoptawidget/index.md

Description of issue: Update Adopt a Widget contribution list. Several PRs were created and merged after the last update of the list which is needed to be added.

### PR DESCRIPTION
**Pull request by sidrao2006**
Fixes #5286.

Changes proposed in this pull request:

*  Add the list of remaining contributions
* Update order

### CODE REVIEW

File: src/adoptawidget/index.md [hunk 1]
```diff
@@ -21,23 +21,32 @@ similar in the future, so stay tuned!
...
-- [Abhishek Ghaskata]({{site.github}}/Abhishek01039) contributed to the [Hero]({{site.master-api}}/flutter/widgets/Hero-class.html) ([pull request]({{site.github}}/flutter/flutter/pull/74415))
+- [tsvillain](https://github.com/tsvillain) contributed to the [Tooltip docs](https://master-api.flutter.dev/flutter/material/Tooltip-class.html) ([pull request](https://github.com/flutter/flutter/pull/69518))
+- [Thesmader](https://github.com/Thesmader) contributed to the [SnackBar docs](https://master-api.flutter.dev/flutter/material/SnackBar-class.html) ([pull request](https://github.com/flutter/flutter/pull/69555))
+- [Muskan Jain](https://github.com/Muskan4010) contributed to the [ColorFiltered docs](https://master-api.flutter.dev/flutter/widgets/ColorFiltered-class.html) ([pull request](https://github.com/flutter/flutter/pull/71093))
+- [Sergio Eric](https://github.com/SergioEric) contributed to the [PreferredSizeWidget docs](https://master-api.flutter.dev/flutter/widgets/PreferredSizeWidget-class.html) ([pull request](https://github.com/flutter/flutter/pull/69539))
+
```
Comment (Abhishek01039) [inline src/adoptawidget/index.md:49]: Please remove this line

**Comment by Abhishek01039** [comment]
cc @sfshaza2 @johnpryan

File: src/adoptawidget/index.md (hunk 1 above)
Comment (sidrao2006) [reply src/adoptawidget/index.md:49]: Thanks

**Review by johnpryan — APPROVED** [review]
Looks good to me.
cc: @sfshaza2

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Pull request merged**